README.md

```diff
@@ -2,7 +2,7 @@
 
 _Form-amd library_
 
-form.js is a small library to help form manipulations and validation. This library uses [amd](http://en.wikipedia.org/wiki/Asynchronous_module_definition) structure.
```

Comment:
Now that form.js has a "constructor" I think its important to start the documentation with a example of how to initialize the form.js instance.

@tisvasconcelos

Reply:
I added in each the examples

@tcelestino.